DependencyInjection/AvanzuAdminThemeExtension.php

```diff
@@ -25,6 +25,13 @@ public function load(array $configs, ContainerBuilder $container)
         $configuration = new Configuration();
         $config = $this->processConfiguration($configuration, $configs);
 
+        if(array_key_exists("options",$configs) == false)
+        {
+            $config["options"] = array(
```

Comment:
Please use short syntax array. And probably more quick to check with isset($config['options'])